src/SDKs/Storage/Storage.Tests/Tests/StorageAccountTests.cs

```diff
@@ -1288,5 +1288,121 @@ public void StorageAccountCreateWithHttpsOnlyTest()
                 Assert.False(account.EnableHttpsTrafficOnly);
             }
         }
+
+        [Fact]
+        public void StorageAccountCMKTest()
+        {
+            var handler = new RecordedDelegatingHandler { StatusCodeToReturn = HttpStatusCode.OK };
+
+            using (MockContext context = MockContext.Start(this.GetType().FullName))
+            {
+                var resourcesClient = StorageManagementTestUtilities.GetResourceManagementClient(context, handler);
+                var storageMgmtClient = StorageManagementTestUtilities.GetStorageManagementClient(context, handler);
+                var keyVaultMgmtClient = StorageManagementTestUtilities.GetKeyVaultManagementClient(context, handler);
+
+                // Create storage account with hot
+                string accountName = TestUtilities.GenerateName("sto");
+                var rgname = "testcmk3";
```

Comment:
In this test case I used a KeyVault key which is pre-created by using PSH. Because KeyVault data-plane seems use a different authentication method with this test case. This rgname have to relate to that pre-created key vault key in this case.